JASP-Engine/JASP/R/ancova.R

```diff
@@ -648,33 +553,35 @@ Ancova <- function(jaspResults, dataset = NULL, options) {
     return()
   
   contrastContainer <- createJaspContainer(title = gettext("Contrast Tables"))
-  contrastContainer$dependOn(c("contrasts", "contrastAssumeEqualVariance", "confidenceIntervalIntervalContrast", 
+  contrastContainer$dependOn(c("contrasts", "confidenceIntervalIntervalContrast", 
                                "confidenceIntervalsContrast", "customContrasts"))
 
   createContrastTable <- function(myTitle, options, contrastType) {
     
     contrastTable <- createJaspTable(title = myTitle)
-    contrastTable$addColumnInfo(name="Comparison", type="string")
-    contrastTable$addColumnInfo(name="Estimate", type="number")
-    contrastTable$addColumnInfo(name="SE", type="number")
-    
-    dfType <- if (options$contrastAssumeEqualVariance && contrastType != "custom") "integer" else "number"
-    contrastTable$addColumnInfo(name="df", type=dfType)
-    contrastTable$addColumnInfo(name="t", type="number")
-    contrastTable$addColumnInfo(name="p", type="pvalue")
-    
-    contrastTable$showSpecifiedColumnsOnly <- TRUE
+    contrastTable$addColumnInfo(name = "Comparison", type = "string")
+    contrastTable$addColumnInfo(name = "estimate", title=gettext("Estimate"), type = "number")
     
     if (options$confidenceIntervalsContrast) {
       
-      thisOverTitle <- gettextf("%.0f%% CI for Mean Difference", options$confidenceIntervalIntervalContrast * 100)
-      contrastTable$addColumnInfo(name="lwrBound", type = "number", title = gettext("Lower"), overtitle = thisOverTitle, )
-      contrastTable$addColumnInfo(name="uprBound", type = "number", title = gettext("Upper"), overtitle = thisOverTitle)
+      thisOverTitle <- gettextf("%s%% CI for Mean Difference", options$confidenceIntervalIntervalContrast * 100)
+      contrastTable$addColumnInfo(name="lower.CL", type = "number", title = gettext("Lower"), overtitle = thisOverTitle)
```

Comment:
Renaming is fine, but what does "CL" stand for? It's not a Confidence Interval?

Reply:
I guess confidence limit or something? It's how the confint function, applied to class emmGrid, returns the confidence intervals.